src/widgets/rating-menu/__tests__/__snapshots__/rating-menu-test.js.snap

```diff
@@ -52,14 +56,21 @@ exports[`ratingMenu() calls twice ReactDOM.render(<RefinementList props />, cont
           false,
           false,
         ],
+        "templates": Object {
+          "andUp": "& Up",
+          "item": "{{#count}}<a class=\\"{{cssClasses.link}}\\" aria-label=\\"{{value}} & up\\" href=\\"{{href}}\\">{{/count}}{{^count}}<div class=\\"{{cssClasses.link}}\\" aria-label=\\"{{value}} & up\\" disabled>{{/count}}
```

Comment:
looking at this like this again, to me it would make sense if the & up was simply part of the item template. If you want to change item, you likely are fine with writing `& Up` yourself. What do you think? Sure, it's a bit more work if you only wan to change that text, but it's not really that bad probably 🤔

Reply:
Yeah I thought about it. Don't have any metrics on how many people change only the `andUp` template so I don't have any strong opinion.

Reply:
Fair point, inside the next documentation we have the default template part of the snippet so it would even easier to extend. Just c/c the snippet update the label and you're done. WDYT?

Reply:
Let's go for that!